app/database/item_store.go

```diff
@@ -14,29 +14,29 @@ type ItemStore struct {
 	*DataStore
 }
 
-// Visible returns a view of the visible items for the given user
-func (s *ItemStore) Visible(user *User) *DB {
-	return s.WhereItemsAreVisible(user)
+// Visible returns a view of the visible items for the given group
```

Comment:
Converting all these functions from "for a given user" to " for a given participant" would probably have been enough.

Reply:
fixed